src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -169,13 +169,14 @@ public static boolean setupDependencies(ProjectConfiguration configuration) thro
         try (FileInputStream fis = new FileInputStream(new File(nameFile));
              ObjectInputStream ois = new ObjectInputStream(fis)) {
             hashes = (Map<String, String>) ois.readObject();
-        } catch (ClassNotFoundException | IOException e) {}
+        } catch (ClassNotFoundException | IOException e) {
+            e.printStackTrace();
```

Comment:
I think we need to propagate the exception here (original code was wrong).
But that might be better to do in another PR -- it's already good to print the stacktrace instead of ignoring it